lib/galaxy/webapps/galaxy/controllers/visualization.py

```diff
@@ -1015,6 +1017,45 @@ def phyloviz( self, trans, id=None, dataset_id=None, tree_index=0, **kwargs ):
             }
         return trans.fill_template_mako( "visualization/phyloviz.mako", data=data, config=config )
 
+    @web.expose
+    @web.require_login( "run Galaxy Interactive Environments" )
```

Comment:
"run Galaxy Interactive Environments"?
Not sure what the best practise in Galaxy is about API endpoints, but maybe this should be part of the lib/galaxy/api?